cameronRomo/silver-nest
# Profile style

### What’s this PR do?
**Features:**
- Profile now renders on Thumbnail/ profile icon click, and closes on 'X' button click
- Profile view conditional renders full screen, centered, and hides Form and Thumbnails
- Apply for Housing button added to profile
- SilverNest logo added to NavBar

**Fixes:**
- Available residents display on first selection of semester in Form (refactored useEffect in FindARoommate)
- Resident Profile now closes on 'x' click and returns to thumbnail/form display
### Where should the reviewer start?
FindARoommate.js: 38
Thumbnails.js: 64
Profile.js: 59

### How should this be manually tested?
- npm start, navigate to FindARoommate, select a semester
- see available residents populate correctly first time
- click on thumbnail
- see resident profile expand and center
- click 'X' to close and return to thumbnails
### Any background context you want to provide?
n/a
### Screenshots (if appropriate):
![Screen Shot 2021-01-12 at 11 25 56 AM](https://user-images.githubusercontent.com/68264128/104356319-0b51c500-54c9-11eb-8a4f-491369da3a80.png)
![Screen Shot 2021-01-12 at 11 26 17 AM](https://user-images.githubusercontent.com/68264128/104356353-160c5a00-54c9-11eb-97fa-c08524bda735.png)

### Questions:
If we have time to get gridlist tiles to hover, should we delete the profile icon from the thumbnails that is currently being used to indicate profile view?
